android/src/main/java/com/guichaguri/trackplayer/module/MusicModule.kt

```diff
@@ -185,21 +185,42 @@ class MusicModule(private val reactContext: ReactApplicationContext?) :
     fun add(data: ReadableArray?, insertBeforeIndex: Int, callback: Promise) {
         if (verifyServiceBoundOrReject(callback)) return
 
+        val tracks: MutableList<Track> = mutableListOf()
         val bundleList = Arguments.toList(data)
-        val tracks: List<Track> = try {
-            Track.createTracks(
-                reactApplicationContext, bundleList, RatingCompat.RATING_HEART
-            )!!
-
-
-        } catch (ex: Exception) {
-            callback.reject("invalid_track_object", ex)
-            return
+        if (bundleList != null) {
+            for (trackBundle in bundleList) {
+                if (trackBundle is Bundle) {
+                    try {
+                        tracks.add(
+                            Track(reactApplicationContext, trackBundle, RatingCompat.RATING_HEART)
+                        )
```

Comment:
It seems your AS is wrapping code very aggressively. You can fix this by going to Preferences -> Code Style -> Kotlin -> Wrapping and Braces -> set Hard wrap at to 999

Reply:
This is a code style. Let's comment on these when we want to put a practice in place.